ci.tfvars

```diff
@@ -23,9 +24,9 @@ stac_server_inputs = {
   stac_id                                     = "stac-server"
   stac_title                                  = "STAC API"
   stac_title                                  = "A STAC API using stac-server"
-  deploy_cloudfront                           = false
   api_rest_type                               = "EDGE"
   api_method_authorization_type               = "NONE"
+  deploy_cloudfront                           = false
```

Comment:
Just moved this to align with default.tfvars for easier diff-ing. On that note, we should compare the two (ci.tfvars, default.tfvars) in near future and question whether we should make some changes ci changes to better reflect what's usually deployed